src/Equinor.ProCoSys.IPO.Query/GetInvitationsQueries/GetInvitationsForExport/GetInvitationsForExcelQueryHandler.cs

```diff
@@ -201,11 +225,12 @@ private async Task<string> GetPersonNameAsync(Guid? personOid)
                     invitation.CompletedAtUtc,
                     invitation.AcceptedAtUtc,
                     invitation.CreatedAtUtc,
-                    organizer
+                    $"{organizer?.FirstName} {organizer?.LastName}"
```

Comment:
Create a get property in Person class which just returns $"{FirstName} {LastName}". Use it here